lib/slim_lint/linter/empty_lines.rb

```diff
@@ -0,0 +1,23 @@
+module SlimLint
+  # Checks for trailing whitespace.
+  class Linter::EmptyLines < Linter
+    include LinterRegistry
+
+    on_start do |_sexp|
+      dummy_node = Struct.new(:line)
+
+      was_empty = true
+      document.source.lines.each_with_index do |line, i|
```

Comment:
Instead of calling `document.source.lines`, you can call `document.source_lines` directly.